model/ai/inference_endpoint.rb

```diff
@@ -18,7 +19,7 @@ class InferenceEndpoint < Sequel::Model
   semaphore :destroy, :maintenance
 
   def display_location
-    LocationNameConverter.to_display_name(location)
+    location.display_name
```

Comment:
This does a query per call if the association is not already loaded. Similarly to the `Location[]` issue, you need to audit all calls to make sure this doesn't cause N+1 query issues in cases where you are retrieving multiple objects and calling the `location` method on each.  You can generally use `eager(:location)` to fix such N+1 issues.

Similar for all models where you are adding a `location` association and defining a similar method.